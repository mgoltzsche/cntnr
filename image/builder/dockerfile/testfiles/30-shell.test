FROM alpine:3.7

SHELL ["/bin/sh", "-c"]

RUN ["/bin/echo", "hello"]
ENTRYPOINT ["/bin/echo", "hello"]
CMD ["/bin/echo", "hello"]

RUN echo hello
ENTRYPOINT echo hello
CMD echo hello

SHELL ["/bin/sh", "-cx"]

RUN echo hello
ENTRYPOINT echo hello
CMD echo hello

# ASSERT CFG /config/Cmd=[/bin/sh -cx echo hello]