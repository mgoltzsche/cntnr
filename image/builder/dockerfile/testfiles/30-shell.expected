FROM alpine:3.7
RUN "/bin/echo" "hello"
ENTRYPOINT "/bin/echo" "hello"
CMD "/bin/echo" "hello"
RUN "/bin/sh" "-c" "echo hello"
ENTRYPOINT "/bin/sh" "-c" "echo hello"
CMD "/bin/sh" "-c" "echo hello"
RUN "/bin/sh" "-cx" "echo hello"
ENTRYPOINT "/bin/sh" "-cx" "echo hello"
CMD "/bin/sh" "-cx" "echo hello"